adapters/Snap/SnapAdapter/GADMediationSnapRewardedAd.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GADMediationSnapRewardedAd : NSObject <GADMediationRewardedAd>
```

Comment:
Nit: Can rename to `GADMediationSnapRewarded` to be consistent with the other classes, or rename:
- `GADMediationSnapInterstitial` to `GADMediationSnapInterstitialAd`
- `GADMediationSnapBanner` to `GADMediationSnapBannerAd`